### ISSUE TITLE
Redundant .asAString() needed in  Ensure.thatTheAnswerTo()

### ISSUE THREAD

**Issue by nbarrett** [serenity-bdd/serenity-core#1746]
I'm beginning to experiment with the new `Ensure` [API for fluent assertions](https://serenity-bdd.github.io/theserenitybook/latest/serenity-screenplay-ensure.html), which is a truly inspirational piece of work, so well done for creating this. 👍 

I have an initial question however - when I  use `Ensure.thatTheAnswerTo()` and pass `Question<String> currentPageId` to it, I get a compilation error with this syntax:

```
 @Override
    public <T extends Actor> void performAs(T actor) {
        actor.attemptsTo(
            Ensure.thatTheAnswerTo(currentPageId())
                .doesNotContain("error"));
    }
```
The only way I can get it to compile is to add `.asAString()` before the assertion, but this seems redundant. Am I doing something wrong?

**Comment by wakaleo** [comment]
There might be a signature missing in the Kotlin classes - the best would be to have a play round with the Kotlin unit tests and add a scenario with this case.

**Comment by wakaleo** [comment]
The fix will be released in version 2.0.62. (There might be a better Kotlin implementation, but I don't have enough experience with Kotlin generics yet to be sure).

**Comment by nbarrett** [comment]
Thanks! this is confirmed to be fixed in 2.0.63 👍
